bitbots_local_planner/src/bitbots_local_planner.cpp

```diff
@@ -84,23 +84,19 @@ namespace bitbots_local_planner
         tf2::fromMsg(current_pose_gm, current_pose);
 
         // Calculate the heading angle from our current position to the carrot
-        double walk_angle = std::fmod(
-            std::atan2(
-                goal_pose_.getOrigin().y() - current_pose.getOrigin().y(),
-                goal_pose_.getOrigin().x() - current_pose.getOrigin().x()),
-            2 * M_PI);
+        double walk_angle = std::atan2(
```

Comment:
Afaik walk angle can be negative after this change. The interval before was [0, 2pi] and now it is [-pi, pi]. But I think we should be fine.

Reply:
That is actually not the case, leaving `fmod` out does not change a thing. `fmod` guarantees the results to have the same sign as the input, and the output of `atan2` is in [-pi, pi]. So the output of `fmod` in this case is also in [-pi, pi], which is just the identity.